docs/hydra/guides/updating-claims-at-refresh.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: claims-at-refresh
+title: Updating claims at token refresh
+---
+
+## Overview
```

Comment:
We usually don't have a first headline, becase you get a H1 directly followed by a H2 when the docs render, see:

# Updating claims at token refresh
## Overview
The Hydra OAuth2 and OpenID Connect server comes with a mechanism that allows updating `id_token` and `access_token` 
 when a registered client sends a token refresh request. The flow is realized by calling the defined refresh token hook endpoint 
 which returns updated data. 

---

```suggestion
```